### PR TITLE
[EMCS-263] Add query parameters for updatedSince on GET endpoints

### PR DESCRIPTION
**Pull request by mpbostock**
Also update the API spec with what's been implemented and minor tidy ups of API spec

### CODE REVIEW

File: resources/public/api/conf/1.0/application.yaml [hunk 1]
```diff
@@ -166,20 +173,13 @@ paths:
...
-        + SubmitCancellation
-        + SubmitChangeOfDestination
-        + SubmitAlertOrRejectionMovement
-        + SubmitSplitMovement
-        + SubmitReasonForShortage
-        
         The **X-Message-Type** header **must** be the same as the message body e.g. for an **IE818 (Record of receipt) message** the **X-Message-Type** header also needs to be **IE818**
         
-        **Note:** The consignor ID or consignee ID must match an excise reference number (ERN) in the bearer token that you use for API authentication. If it does not, a 403 Forbidden error will be thrown.
+        **Note:** The consignor ID or consignee ID in the message must match an excise reference number (ERN) in the bearer token that you use for API authentication. If it does not, a 403 Forbidden error will be thrown.
```
Comment (ChrisCookOC) [inline resources/public/api/conf/1.0/application.yaml:178]: Not sure how best to word this but 813 and 810 don't have the consignor or (current) consignee in the message itself, so we validate it against the value from the database.

Comment (mpbostock) [reply resources/public/api/conf/1.0/application.yaml:178]: These messages will have a ARC though right? We should be able to pull out the movement using the ARC and then check if one of their ERNs match what we have recorded in the movement (I know there is a question around IE813 and whether that means we either add an additional consignee to the movement that needs addressing too)

Comment (ChrisCookOC) [reply resources/public/api/conf/1.0/application.yaml:178]: yeah that's what we do, take the arc, look up consignor/consignee in database.

Comment (mpbostock) [reply resources/public/api/conf/1.0/application.yaml:178]: OK to leave this as-is? Hopefully the general idea is there, if some messages don't have the consignor/consignee not sure of a better way to explain

**Review by ChrisCookOC — APPROVED** [review]

**Review by ChrisCookOC — APPROVED** [review]

**Pull request merged**